components/property_info.rst

```diff
@@ -409,6 +409,22 @@ constructor. It supports return and scalar types for PHP 7.
     // Initializable information
     $reflectionExtractor->isInitializable($class, $property);
 
+Excluding static properties
+...........................
+By default, the static properties are extracted. To get them excluded,
+you can use the ``exclude_static_properties`` context option.
+
+
+.. code-block:: php
+
+    use Symfony\Component\PropertyInfo\Extractor\ReflectionExtractor;
+
+    $reflectionExtractor = new ReflectionExtractor();
+
+    $reflectionExtractor->getProperties($class, [
```

Comment:
code example could maybe be reduced to this single call, given the context above?